server/config/subject_page.proto

```diff
@@ -261,6 +261,40 @@ message MapTileSpec {
   string geo_json_prop = 3;
 }
 
+// Spec for getting value from property values api call
+message PropertySpec {
+  string property = 1;
+  string direction = 2;
+  string type = 3;
+}
+
+// Spec for just displaying a value
+message DisplayValueSpec {
+  repeated string value = 1;
+  repeated string source = 2;
+}
+
+message AnswerMessageTileSpec {
+  oneof answer {
+    PropertySpec property = 1;
+    DisplayValueSpec display_value = 2;
+  }
+  string entity = 3;
+}
+
+message AnswerTableTileSpec {
+  message TableColumn {
+    string header = 1;
+    PropertySpec property = 2;
+  }
+  repeated TableColumn column = 1;
+  repeated string entity = 2;
+}
+
```

Comment:
Please describe the spec, how this matches to a table.  And is "entity" required or optional?

Reply:
done